examples/domain_reduction.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Can we add a link to this notebook to the **Quick Start** list in the `README.md`?